packages/coding-standard/tests/Rules/PrefferedMethodCallOverFuncCallRule/config/configured_rule.neon

```diff
@@ -0,0 +1,10 @@
+services:
+    - Symplify\CodingStandard\PhpParser\NodeNameResolver
+
+    -
+        class: Symplify\CodingStandard\Rules\PrefferedMethodCallOverFuncCallRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            funcCallToPrefferedStaticCalls:
+                file_get_contents: ['Symplify\SmartFileSystem\SmartFileSystem', 'readFile']
+                substr: ['Symplify\CodingStandard\Tests\Rules\PrefferedMethodCallOverFuncCallRule\Fixture\SkipSelfCall', 'run']
```

Comment:
:+1:

Reply:
Could you put the `file_get_contents` and other similar `SmartFileSystem` methods to `symplify-rules.neon`? So we have it covered

Reply:
added